contracts/base/token/StandardToken.sol

```diff
@@ -16,15 +16,15 @@ contract StandardToken is ERC20 {
 
     mapping(address => uint256) balances;
 
-    uint256 totalSupply;
+    uint256 totalSupply_;
```

Comment:
instead of changing this back to a non-standard format, lets change the variable all together to something else that doesn't conflict with ERC20 namespaces.